lib/ansible/galaxy/api.py

```diff
@@ -233,11 +234,17 @@ def __init__(self, namespace, name, version, download_url, artifact_sha256, depe
         self.dependencies = dependencies
 
 
+@functools.total_ordering
```

Comment:
total_ordering isn't present in 2.7 so you can't use this until ansible-2.12.

Reply:
Not technically.  We do not support 2.6 on the controller, and as long as it isn't causing testing issues, I see no reason why not.  2.7 is our minimum required for the controller.

Reply:
Ah, sorry.  I checked the python 3 five and saw it wasn't present from the outset.  I forgot that puthon2.7's release can't after python 3.1 so total_ordering might have been added to both.